pack/side/film_fatale_encounter.json

```diff
@@ -0,0 +1,1207 @@
+    "clues": null,
+    "code": "72026",
+    "double_sided": true,
+    "encounter_code": "film_fatale_space",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "The telescope's transporting beam may be able to send the Saturnites home, but the monarch is hellbent upon taking it.",
+    "illustrator": "Drazenka Kimpel",
+    "name": "From Whence They Came",
+    "pack_code": "film_fatale",
+    "position": 26,
+    "quantity": 1,
+    "stage": 3,
+    "text": "Each Possessed Extra gets +2 health, +1 fight, +1 evade, and gains elusive and the [[Saturnite]] trait.\n[action] Spend 1 [per_investigator] clues, as a group: Place 1 shard on Helios Telescope.\n<b>Forced</b> - If Helios Telescope has no shards on it, advance.\n<b>Objective</b> - Dethrone the monarch. If the Saturnite Monarch is defeated, advance.",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "The looming walls spring forth from the desert, grasping for the heavens. They're accompanied by floating glass orbs resembling watchful eyes",
+    "back_traits": "Cosmos.",
+    "clues": 1,
+    "code": "72027",
+    "double_sided": true,
+    "encounter_code": "film_fatale_space",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "High Ruler's Bastion",
+    "pack_code": "film_fatale",
+    "position": 27,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "<b>Forced</b> - At the end of the round; if there are no clues on High Ruler's Bastion: Place clues on this location up to its clue value.\n[free] If the Saturnite Moanarch is in the victory display: Place 1 shard on Helios Telescope. (Max once per game.)",
+    "traits": "Cosmos.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "A narrow passage winds beneath two looming cliffs. You're convinced someone-or something-is watching.",
+    "back_traits": "Cosmos.",
+    "clues": 4,
+    "clues_fixed": true,
+    "code": "72028",
+    "double_sided": true,
+    "encounter_code": "film_fatale_space",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Teetawn Passage",
+    "pack_code": "film_fatale",
+    "position": 28,
+    "quantity": 1,
+    "shroud": -14,
```

Comment:
We don't have "shroud per investigator" implemented. The one other location that uses this has "-11" to represent "1 per investigator" for shroud.